pyproject.toml

```diff
@@ -30,28 +28,29 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
+python = "^3.8.1"
 SQLAlchemy = ">=1.4.17,<=1.4.41"
 pydantic = "^1.8.2"
 sqlalchemy2-stubs = {version = "*", allow-prereleases = true}
 
 [tool.poetry.dev-dependencies]
-pytest = "^7.0.1"
-mypy = "0.971"
-flake8 = "^5.0.4"
-black = {version = "^22.10.0", python = "^3.7"}
-mkdocs = "^1.2.1"
-mkdocs-material = "^8.1.4"
-pillow = {version = "^9.3.0", python = "^3.7"}
-cairosvg = {version = "^2.5.2", python = "^3.7"}
-mdx-include = "^1.4.1"
-coverage = {extras = ["toml"], version = "^6.2"}
-fastapi = "^0.68.1"
-requests = "^2.26.0"
-autoflake = "^1.4"
-isort = "^5.9.3"
-async_generator = {version = "*", python = "~3.6"}
-async-exit-stack = {version = "*", python = "~3.6"}
+pytest = "^7.2.0"
+mypy = "0.991"
+flake8 = "^6.0.0"
+black = {version = "^22.12.0", python = "^3.8"}
+mkdocs = "^1.4.2"
+mkdocs-material = "^8.5.11"
+pillow = {version = "^9.3.0", python = "^3.8"}
+cairosvg = {version = "^2.5.2", python = "^3.8"}
+mdx-include = "^1.4.2"
+coverage = {extras = ["toml"], version = "^6.5"}
+fastapi = "^0.88.0"
+requests = "^2.28.0"
+httpx = "^0.23.1"
+autoflake = "^2.0.0"
+isort = "^5.11.3"
+async_generator = {version = "*"}
+async-exit-stack = {version = "*"}
```

Comment:
I don't think these two dependencies are needed anymore since they provide features included in Python 3.7, so they were only used to give Python 3.6 support.

Reply:
I had this point pending. Now this two deps are deprecated.